sail.md

```diff
@@ -246,6 +247,14 @@ Your application's `docker-compose.yml` file also contains an entry for a [Redis
 
 To connect to your application's Redis database from your local machine, you may use a graphical database management application such as [TablePlus](https://tableplus.com). By default, the Redis database is accessible at `localhost` port 6379.
 
+
+<a name="redis"></a>
+### MongoDB
+
+Your application's `docker-compose.yml` file also contains an entry for a [MongoDB Community](https://www.mongodb.com) container. This container uses a [Docker volume](https://docs.docker.com/storage/volumes/) so that the data stored in your MongoDB data is persisted even when stopping and restarting your containers. Once you have started your containers, you may connect to the Redis instance within your application by setting your `REDIS_HOST` environment variable within your application's `.env` file to `redis`.
```

Comment:
Seems like a typo - need to get rid of redis 
> connect to the Redis instance within your application by setting your `REDIS_HOST` environment variable within your application's `.env` file to `redis`.
> 
> To connect to your application's Redis database from your local machine, you may use a graphical database management application such as [TablePlus](https://tableplus.com). By default, the Redis database is accessible at `localhost` port 6379.

Reply:
And, We should mention compass instead of TablePlus
https://www.mongodb.com/docs/compass/current/

Reply:
Removed the sail modification, it should be added after https://github.com/GromNaN/laravel-sail/pull/1